components/application-configuration/ui/src/main/resources/PhenoTips/DBConfigurationClass.xml

```diff
@@ -67,7 +67,7 @@ $!value
       <name>dateOfBirthFormat</name>
       <number>5</number>
       <picker>0</picker>
-      <prettyName>Date of birth format</prettyName>
+      <prettyName>$services.localization.render('phenotips.DBConfigurationClass.dateOfBirthFormat')</prettyName>
```

Comment:
Nope, leave the original text here, and add the following translation key:

    PhenoTips.DBConfigurationClass_dateOfBirthFormat=Date of birth format